framework/source/class/qx/ui/table/pane/Pane.js

```diff
@@ -655,6 +661,46 @@ qx.Class.define("qx.ui.table.pane.Pane",
       this.fireEvent("paneUpdated");
     },
 
+    _updateSingleRow: function(row) {
+      var elem = this.getContentElement().getDomElement();
+      if (!elem || !elem.firstChild) {
+        // pane has not yet been rendered, just exit
+        return;
+      }
+      var visibleRowCount = this.getVisibleRowCount();
+      var firstRow = this.getFirstVisibleRow();
+
+      if (row < firstRow || row > firstRow+visibleRowCount) {
+        // No need to redraw it
+        return;
+      }
+
+      var modelRowCount = this.getTable().getTableModel().getRowCount();
+
+      var tableBody = elem.firstChild;
+      var tableChildNodes = tableBody.childNodes;
+      var offset = row - firstRow;
+      var rowElem = tableChildNodes[offset];
+
+      if (row > modelRowCount || rowElem === undefined) {
```

Comment:
Since we're still supporting old browser versions where `undefined` could be overwritten (e.g., `var undefined = 23;`), I suggest replacing `rowElem === undefined` with `typeof rowElem == "undefined"`